includes/admin/class.llms.admin.assets.php

```diff
@@ -412,6 +412,8 @@ public function admin_print_scripts() {
 		';
 
 		echo '<script type="text/javascript">window.LLMS = window.LLMS || {};</script>';
+		
+		// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- get_js_strings returns json_encoded strings.
 		echo '<script type="text/javascript">window.LLMS.l10n = window.LLMS.l10n || {}; window.LLMS.l10n.strings = ' . LLMS_L10n::get_js_strings( true ) . ';</script>';
 
```

Comment:
Could we not use `esc_js` here?

```suggestion
		echo '<script type="text/javascript">window.LLMS.l10n = window.LLMS.l10n || {}; window.LLMS.l10n.strings = \'' . esc_js( LLMS_L10n::get_js_strings( true ) ) . '\';</script>';

```

Reply:
LLMS_L10n::get_js_strings returns an array of strings that are already escaped.

Reply:
It's not already escaped in this case and we're using `json_encode` vs `wp_json_encode` so it could have output we don't want.

Reply:
So you can't use `esc_js()` on either the result of `json_encode()` or `wp_json_encode()` as it'll turn the double quotes into `&quot;`. The docs aren't too clear on that.

What we can do instead is remove the `true` to return json, and do the `wp_json_encode` here instead. That should play nice with the scanner.

Reply:
Verified that `wp_json_encode`-ing here works and scanner doesn't complain.